packages/core/src/server-runtime-client.ts

```diff
@@ -25,6 +25,9 @@ import { resolvedSyncPromise } from './utils-hoist/syncpromise';
 import { _INTERNAL_flushLogsBuffer } from './logs/exports';
 import { isPrimitive } from './utils-hoist';
 
+// TODO: Make this configurable
+const DEFAULT_LOG_FLUSH_INTERVAL = 5000;
```

Comment:
This also exists in the browser SDK.

I'm going to create another PR after this that unifies this as a logging option, and update the develop docs as well.